app/javascript/dashboard/components/widgets/conversation/components/SLACardLabel.vue

```diff
@@ -93,8 +97,20 @@ export default {
   },
   mounted() {
     this.updateSlaStatus();
+    this.createTimer();
+  },
+  beforeDestroy() {
+    if (this.timer) {
+      clearTimeout(this.timer);
+    }
   },
   methods: {
+    createTimer() {
+      this.timer = setTimeout(() => {
```

Comment:
I am assuming there was no straightforward way to handle this outside the component, can you see if this works properly for ~100 conversations let me know if you need help generating those

Reply:
We are using the virtual list for the conversation list. I think it won't affect the performance.